staging/src/k8s.io/api/README.md

```diff
@@ -1 +1,20 @@
-This repo is still in the experimental stage. Shortly it will contain the schema of the API that are served by the Kubernetes apiserver.
+# api
+
+Schema of the external API types that are served by the Kubernetes apiserver.
+
+## Purpose
+
+This repo is still in the experimental stage.
+
+Canonical location of the Kubernetes API definition. Most likely interaction with this repository is as a dependency of client-go.
+
+## Compatibility
+
+There are *NO compatibility guarantees* for this repository. It is in direct support of Kubernetes, so branches
+will track Kubernetes and be compatible with that repo. As we more cleanly separate the layers, we will review the
+compatibility guarantee.
+
+## Where does it come from?
+
+`api` is synced from https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api.
+Code changes are made in that location, merged into `k8s.io/kubernetes` and later synced here.
```

Comment:
Maybe we could add something like https://github.com/kubernetes/apimachinery#things-you-should-not-do below this?

Reply:
👍 Added one thing, any other suggestions what we should add there?